src/components/Totals.js

```diff
@@ -2,47 +2,49 @@ import { Component } from 'preact';
 import { html } from 'htm/preact';
 
 export class Totals extends Component {
+  constructor() {
+    super();
+    this.setState({
```

Comment:
you can set default state as:

```js
this.state = {
  items: []
}
```

iirc calling setstate in the constructor will do unnecessary work because the component hasnt rendered yet anyway